lib/p2p/packets/PacketType.ts

```diff
@@ -1,3 +1,21 @@
+enum PacketType {
+  Hello = 0,
```

Comment:
I think it would make sense to explicitly number each packet instead of just the first. That way if we ever delete a packet type, the numbering of the other packets won't change.